plugin/flavor/zookeeper/flavor.go

```diff
@@ -167,7 +167,7 @@ func (s zkFlavor) Prepare(
 			return spec, errors.New("Manager nodes require an assigned logical ID")
 		}
 
-		spec.Init = generateInitScript(properties.UseDocker, allocation.LogicalIDs, *spec.LogicalID)
+		spec.Init += "\n" + generateInitScript(properties.UseDocker, allocation.LogicalIDs, *spec.LogicalID)
```

Comment:
Why a newline here -- does this depend on how this flavor is used (by itself or in a combo)?